monitoring/local_monitoring/local_monitoring_demo.sh

```diff
@@ -129,139 +138,36 @@ function kruize_local() {
-  	echo
-
-  	echo
-  	echo "############################################################"
-  	echo "#     Display metadata for test-multiple-import namespace"
-  	echo "############################################################"
-  	echo
-  	curl "http://${KRUIZE_URL}/dsmetadata?datasource=${DATASOURCE}&cluster_name=${CLUSTER_NAME}&namespace=test-multiple-import&verbose=true"
-  	echo
-
-  	echo
-  	echo "######################################################"
-  	echo "#     Delete previously created experiment"
-  	echo "######################################################"
-  	echo
-  	echo "curl -X DELETE http://${KRUIZE_URL}/createExperiment -d @./create_tfb_exp_multiple_import.json"
-  	curl -X DELETE http://${KRUIZE_URL}/createExperiment -d @./create_tfb_exp_multiple_import.json
-  	echo "curl -X DELETE http://${KRUIZE_URL}/createExperiment -d @./create_tfb-db_exp_multiple_import.json"
-  	curl -X DELETE http://${KRUIZE_URL}/createExperiment -d @./create_tfb-db_exp_multiple_import.json
-  	echo
-
-  	echo
-  	echo "######################################################"
-  	echo "#     Create kruize experiment"
-  	echo "######################################################"
-  	echo
-  	echo "curl -X POST http://${KRUIZE_URL}/createExperiment -d @./create_tfb_exp_multiple_import.json"
-  	curl -X POST http://${KRUIZE_URL}/createExperiment -d @./create_tfb_exp_multiple_import.json
-  	echo "curl -X POST http://${KRUIZE_URL}/createExperiment -d @./create_tfb-db_exp_multiple_import.json"
-  	curl -X POST http://${KRUIZE_URL}/createExperiment -d @./create_tfb-db_exp_multiple_import.json
-  	echo
-
-	echo "Sleeping for 3mins before generating the recommendations!"
-	sleep 3m
+	apply_benchmark_load ${APP_NAMESPACE}
 
   	echo
   	echo "######################################################"
   	echo "#     Generate recommendations for every experiment"
   	echo "######################################################"
   	echo
-	curl -X POST "http://${KRUIZE_URL}/generateRecommendations?experiment_name=monitor_tfb_benchmark"
-	curl -X POST "http://${KRUIZE_URL}/generateRecommendations?experiment_name=monitor_tfb-db_benchmark"
-  	curl -X POST "http://${KRUIZE_URL}/generateRecommendations?experiment_name=monitor_tfb_benchmark_multiple_import"
-  	curl -X POST "http://${KRUIZE_URL}/generateRecommendations?experiment_name=monitor_tfb-db_benchmark_multiple_import"
-  	echo ""
+	tfb_recommendations=$(curl -X POST "http://${KRUIZE_URL}/generateRecommendations?experiment_name=monitor_tfb_benchmark")
+	echo $tfb_recommendations
+	tfb_db_recommendations=$(curl -X POST "http://${KRUIZE_URL}/generateRecommendations?experiment_name=monitor_tfb-db_benchmark")
+	echo $tfb_db_recommendations
+
+	if [ -z "$tfb_recommendations" ] || [ -z "${tfb_db_recommendations}" ]; then
+		echo ""
+		echo "######################################################"
+		echo "RECOMMENDATIONS MIGHT BE EMPTY IF THERE IS NO ENOUGH DATA AVAILABLE!"
```

Comment:
Please be specific about the minimum amount of data required. In our case we need atleast 2 data points to generate a recommendation

Reply:
Done.